spec/presenters/finder_presenter_spec.rb

```diff
@@ -134,6 +134,13 @@
         presenter.atom_url.should == "/mosw-reports.atom?format=publication&keyword=legal&state=open"
       end
     end
+
+    context "when the finder is ordered by title" do
+      it "doesn't return an atom_url" do
+        presenter.stub(:default_order).and_return('title')
```

Comment:
I'd prefer to avoid stubbing a method on the object we're testing. Can you make the content item respond with the correct value instead?